src/ios/Accessibility.ts

```diff
@@ -89,6 +89,11 @@ export class Accessibility extends NativeAccessibility {
             }
         }
     }
+
+    private compareRawAnnouncements(postedAnnouncement: string, payloadAnnouncement: string): boolean {
```

Comment:
I think need to use `_` before name `_compareRawAnnouncements`, just like other private methods

Reply:
Agree, fixed